app/src/test/java/com/ikarabulut/ComeAliveRandomlyTest.java

```diff
@@ -0,0 +1,57 @@
+package com.ikarabulut;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class ComeAliveRandomlyTest {
+    private ComeAliveRandomly gameRules;
+
+    @BeforeEach
+    void initializeRulesClass() {
+        gameRules = new ComeAliveRandomly();
+    }
+
+    @Test
+    @DisplayName("When an Alive Cell has 0 alive neighbors passed, it is underpopulated, meaning it will not survive for the next generation")
+    void aliveCheck_0Passed() {
+        int numOfAliveNeighborsPassed = 0;
+        boolean isAlive = true;
+        boolean test = gameRules.willBeAliveNextGeneration(numOfAliveNeighborsPassed, isAlive);
```

Comment:
Use a more descriptive name for the variable here, like `willBeAlive`

Same for the other tests too.

Reply:
The `isAlive` is the argument passed to `willBeAliveNextGeneration()` letting the method know if the cell being run through the rules is alive or dead.

If you're confused that means the naming convention is off though. So Ill update it, but `willBeAlive` won't match up with what it is doing.

Reply:
Oh wait you meant for `test`. Yeah that makes sense now